src/rust/engine/fs/store/src/lib.rs

```diff
@@ -73,16 +70,9 @@ use workunit_store::{in_workunit, Level, Metric};
 
 use crate::remote::ByteStoreError;
 
-const KILOBYTES: usize = 1024;
-const MEGABYTES: usize = 1024 * KILOBYTES;
+const MEGABYTES: usize = 1024 * 1024;
 const GIGABYTES: usize = 1024 * MEGABYTES;
 
-/// How big a file must be to become an immutable input symlink.
-// NB: These numbers were chosen after micro-benchmarking the code on one machine at the time of
-// writing. They were chosen using a rough equation from the microbenchmarks that are optimized
-// for somewhere between 2 and 3 uses of the corresponding entry to "break even".
-const IMMUTABLE_FILE_SIZE_LIMIT: usize = 512 * KILOBYTES;
```

Comment:
Instead of throwing out the baby with the bathwater, can we instead just put this to `usize` max and file a ticket?

I'm very very very afraid of merge conflicts when I need to put this back.

Reply:
Additionally, that change might re-create flakes, which is a clue in itself.

Reply:
@thejcannon Can you use the suggest edits feature to propose that here? I'm not sufficiently au fait with rust to get that right without looking at docs.

Reply:
OK, I think I can do something that'll work. One moment.